### ISSUE TITLE
Empty cfntools files in instance generated with usability branch

### ISSUE THREAD

**Issue by tomassedovic** [sdake/heat-jeos#11]
Steps:

Checkout the `usability` branch.

Build the image:

```
sudo -E ./bin/heat-jeos -d -y create F16-x86_64-cfntools-jeos --register-with-glance
```

Launch a stack:

```
./bin/heat -d create wordpress --template-file=templates/WordPress_Single_Instance.template --parameters="InstanceType=m1.xlarge;DBUsername=${USER};DBPassword=verybadpass;KeyName=${USER}_key"
```

SSH into the stack.

Look at the `/opt/aws/bin/*` files. They are all empty.

**Comment by tomassedovic** [comment]
The bug:

This generates the `jeos_guest` object that is later passed to `build_jeos` for image creation.

https://github.com/heat-api/heat-jeos/blob/usability/bin/heat-jeos#L111

However, this `jeos_guest` gets passed the source template, not the one with cfntools file contents included.

I'll fix and test this on Tuesday 10th unless some gets to it before. Should be an easy fix.

**Comment by tomassedovic** [comment]
Fixed in commit 7b51f55
